app/src/main/java/app/insti/fragment/ComplaintDetailsFragment.java

```diff
@@ -195,16 +195,16 @@ private void populateViews() {
                 textViewStatus.setBackgroundTintList(ColorStateList.valueOf(getContext().getResources().getColor(R.color.colorGreen)));
                 textViewStatus.setTextColor(getContext().getResources().getColor(R.color.secondaryTextColor));
             }
+            addVotesToView(detailedComplaint);
+            addCommentsToView(detailedComplaint);
+            initViewPagerForImages(detailedComplaint);
```

Comment:
Please don't rearrange stuff in a feature PR

Reply:
This was done due to the following sequence of events:
-- The feature of relevant complaints that I was implementing requires loading multiple complaintDetailsFragment consecutively.
-- On going back to one the complaints from which I originally started, there was bug wherein the upvotes, comments and images were not visible while the tags were. 
-- During debugging I found that these three methods were not getting executed, or to be more precise the execution of the populateViews() stopped after adding tags to the view.

By rearranging the methods, I was able to fix it. Would you like me commit this separately when I'm making a PR for the relevant complaints functionality?

Reply:
I would've let it pass if it's a bug fix, but I don't maintain this repo so I'll leave it to @unstablebrainiac to decide.